implementation/configuration/src/configuration_impl.cpp

```diff
@@ -561,7 +561,7 @@ bool configuration_impl::load_data(const std::vector<configuration_element> &_el
 
         if (is_logging_loaded_) {
             logger::logger_impl::init(shared_from_this());
-            for (auto w : its_warnings)
+            for (auto const& w : its_warnings)
```

Comment:
Prefer `const auto` over `auto const`.

```suggestion
            for (const auto& w : its_warnings)
```